wls-common/security/src/main/java/de/muenchen/oss/wahllokalsystem/wls/common/security/EncryptionBuilder.java

```diff
@@ -19,7 +19,7 @@
 public class EncryptionBuilder {
 
     private static final Logger log = LoggerFactory.getLogger(EncryptionBuilder.class);
-    private static final String technischeExceptionKonstante = "S";
+    private static final String technischeExceptionKonstante = "399";
```

Comment:
bitte die Tests erweitern zur Verifikation des Codes

Reply:
habe jetzt keine Möglichkeit gesehen, den Code selbst auszulesen, also habe ich ihm den Code in der Message mitgegeben

Reply:
du kannst die Property der Exception vergleichen. Eine Property ist der Code.